photoeditor/src/main/java/ja/burhanrashid52/photoeditor/MultiTouchListener.kt

```diff
@@ -126,10 +126,6 @@ internal class MultiTouchListener(
         } ?: false
     }
 
-    fun setOnMultiTouchListener(onMultiTouchListener: OnMultiTouchListener?) {
-        this.onMultiTouchListener = onMultiTouchListener
```

Comment:
This is the only place where `onMultiTouchListener` is set. If this is unused, that means we don't need `onMultiTouchListener` at all. So it's better to remove the variable and its usage inside this class, as well as the logic which is responsible for calling `onMultiTouchListeners`.

Reply:
@benju69 Is this comment resolved?